datacenter/ucp/2.0/guides/content-trust/index.md

```diff
@@ -160,6 +160,12 @@ trusted repository.
 Valid delegation roles take the form of `targets/<delegation>`, where
 `<delegation>` does not include further slashes.
 
+You will need to add the key to at least one delegation in addition to the `targets/releases` delegation in order for UCP to honor the signed content:
+
+```bash
+$ notary delegation add -p <dtr_url>/<account>/<repository> targets/devops --all-paths user1.pem user2.pem
```

Comment:
this might need to be applied more broadly across this doc and others - happy to make this change in a followup PR